core/cas-server-core-authentication/src/main/java/org/apereo/cas/authentication/handler/support/AbstractUsernamePasswordAuthenticationHandler.java

```diff
@@ -58,7 +58,6 @@ protected HandlerResult doAuthentication(final Credential credential) throws Gen
         }
 
         userPass.setUsername(transformedUsername);
-        userPass.setPassword(this.passwordEncoder.encode(userPass.getPassword()));
```

Comment:
We cannot encode it using passwordEncoder (at least in case of ** BCryptPasswordEncoder** is used as passwordEncoder ) . passwordEncoder.matches require rawPassword to go as first parameter therefore we cannot encoded